classes/security/Validation.inc.php

```diff
@@ -397,18 +392,42 @@ static function isLoggedInAs() {
 	 * Shortcut for checking authorization as site admin.
 	 * @return boolean
 	 */
-	static function isSiteAdmin() {
+	public static function isSiteAdmin() {
 		return self::isAuthorized(ROLE_ID_SITE_ADMIN);
 	}
 
+	/**
+	 * Check if the user is enable
+	 * 
+	 * @param 	$user 		User user to register in the session
+	 * @param 	$reason 	string reference to string to receive the reason an account was disabled; null otherwise
+	 * 
+	 * @return 	bool 		true to identify user is enable and false otherwise
+	 */
+	public static function isEnable($user, &$reason) {
```

Comment:
I don't think I see the value in this extra function. "Helper" classes like `Validation` are messy and we're trying to decrease our reliance on them; this function is essentially just `$user->getDisabled()` followed by `$user->getDisabledReason()`, with `$reason` not used in all calling cases.